CHANGELOG.md

```diff
@@ -72,8 +72,20 @@
 
   ([PR #969](https://github.com/alphagov/govuk-frontend/pull/969))
 
+- Remove the width declaration from the `<select>` component
+
+  The `<select>` component’s width will now be defined by it’s content. This addresses some accessibility issues with the select being 100% wide by default. If you want to style your select to be 100% wide we have added a new override class to allow this.
```

Comment:
Non-blocking it would be good to explicitly reference the new class that can be used, not the end of the world though.